src/data/migrations/20210626150622_create-meme-table.ts

```diff
@@ -0,0 +1,16 @@
+import { Knex } from "knex";
```

Comment:
Still weird file name `20210626150622_create-meme-table.ts`?

Reply:
This is needed for knex to potentially upgrade the database, if we change the schema. I think there should be a better way, but right now we need to live with it 🤷